src/PageContent/RequestPickup/PickupSummary.js

```diff
@@ -104,9 +104,9 @@ class PickupSummary extends React.Component {
                                     <div>
                                         <p id="name">{this.props.dgRequested.name}</p>
                                         <div className="contact">
-                                            <p>{this.props.dgRequested.coordinator.name}</p>
-                                            <p>{this.props.dgRequested.coordinator.phone}</p>
-                                            <p>{this.props.dgRequested.coordinator.email}</p>
+                                            <p>{this.props.dgRequested.primaryContact.name}</p>
```

Comment:
nit: this would be a good place to unpack props, or parts of props:
```
const { primaryContact } = this.props.dgRequested;
...
<p>{primaryContact.name}</p>
```